README.md

```
@@ -558,7 +497,434 @@ flowchart TB
 
 ## 📂 패키지 구조
 
-> 🚧 개발 완료되면 상세 패키지 구조 업데이트
+### 전체 프로젝트 구조
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)